src/scales/scale.time.js

```diff
@@ -404,21 +404,39 @@ function computeOffsets(table, ticks, min, max, options) {
 	return {start: start, end: end};
 }
 
+function setMajorTicks(scale, indices, ticks, majorUnit) {
+	var adapter = scale._adapter;
+	var first = +adapter.startOf(ticks[0].value, majorUnit);
+	var last = ticks[ticks.length - 1].value;
+	var major, index;
+
+	for (major = first; major <= last; major = +adapter.add(major, 1, majorUnit)) {
+		index = indices[major];
+		if (typeof index !== 'undefined') {
```

Comment:
`if (index || index === 0) {` would save 13 bytes 📦

Reply:
I changed it to `!isNaN(index)` to make it shorter while keeping it at just one condition (vs. having the `||`)

Reply:
Because `isNaN` is very slow, `index >= 0` would be better.
https://jsperf.com/nan-vs-num-comp

Reply:
done